packages/web/components/Setup/SetupUsername.tsx

```diff
@@ -33,14 +35,16 @@ export const SetupUsername: React.FC<SetupUsernameProps> = ({ username, setUsern
       if (error.message.includes('Uniqueness violation')) {
         errorDetail = 'This username is already taken 😢';
       } else if (error.message.includes('username_is_valid')) {
-        errorDetail = 'A username can only contain letters, numbers, and dashes.';
+        errorDetail =
```

Comment:
I try to always have a character that marks the end of an indentation block.

```suggestion
        errorDetail = (
          'A username can only contain letters, numbers, and dashes.'
        );
```